CHANGELOG.md

```diff
@@ -10,6 +10,11 @@
   Rename SwiftLintBuildToolPlugin.
   Add Swift Package Manager installation instructions.  
   [garricn](https://github.com/garricn)
+  
+* Adjusting CodeClimate Reporter output to have lower case severity
+  values for use in Gitlab and other CodeClimate compatible tools can 
+  import the report without modification.
+  [waitButY](https://github.com/waitbutY)
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown. Also, the important aspect is compliance with the spec:
```suggestion
* Fix Code Climate reporter output by having lower case severity
  values to comply with the Code Climate specification.  
  [waitButY](https://github.com/waitbutY)
```